packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -157,17 +156,19 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
                             ['ms-DatePicker-day--today ' + styles.dayIsToday]: day.isToday,
                             ['ms-DatePicker-day--highlighted ' + styles.dayIsHighlighted]: day.isSelected
                           }) }
-                        role='gridcell'
+                        role='link'
+                        href='#'
```

Comment:
Also, this should probably be a button since it's not doing any navigation and is same page.